src/server/templates/_components/form-nhs-number-radio.njk

```diff
@@ -40,7 +40,7 @@
     </legend>
   {% endif %}
 
-  <div class="multiple-choice__container">
+  <div class="multiple-choice__container margin-btm20">
```

Comment:
Classes to apply specific amounts of spacing should ideally be avoided. Spacing between elements has been handled by the `* + *` selector to offer consistency but it can also be overridden in specific cases if necessary.